install.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+url="https://raw.githubusercontent.com/CFPB/open-source-project-template/master"
+
+root_files=(
+  ".gitattributes"
+  ".gitignore"
+  "CHANGELOG.md"
+  "CONTRIBUTING.md"
+  "INSTALL.md"
+  "LICENSE"
+  "README.md"
+  "TERMS.md"
+)
+
+gh_files=(
+  "ISSUE_TEMPLATE.md"
+  "PULL_REQUEST_TEMPLATE.md"
+)
+
+echo "creating .github directory"
+
+mkdir -p .github
+
+echo "downloading files"
+
+for file in "${root_files[@]}"; do
+  if [ ! -f "./${file}" ]; then
```

Comment:
@willbarton mentioned using a script to update these files in a repo that already has them. This condition would not download a file if it already exists, correct?

Do we want to address the updating use case now, or in a later iteration?